include/mbedtls/platform.h

```diff
@@ -45,6 +45,7 @@
 
 #define MBEDTLS_ERR_PLATFORM_HW_ACCEL_FAILED     -0x0070 /**< Hardware accelerator failed */
 #define MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED -0x0072 /**< The requested feature is not supported by the platform */
+#define MBEDTLS_ERR_PLATFORM_FAULT_DETECTED      -0x0071 /**< A fault was detected in a critical path, likely indicative of an active physical attack */
```

Comment:
Good point, and thanks for suggesting a better wording - I'll use that.

It could also be a real programmer using their favourite editor https://xkcd.com/378/ ;)